app/assets/javascripts/discourse/app/components/filter-input.gjs

```diff
@@ -7,7 +7,7 @@ import concatClass from "discourse/helpers/concat-class";
 import noop from "discourse/helpers/noop";
```

Comment:
This dc-filter-input component was used only in chat for the picker, it doesnt make sense to have this in chat anymore, ideally I would prefer we had a more standard solution for this kind of inputs